backend/src/plugins/ModActions/functions/actualKickMemberCmd.ts

```diff
@@ -67,7 +68,8 @@ export async function actualKickMemberCmd(
     return;
   }
 
-  const reason = formatReasonWithAttachments(args.reason, msg.attachments);
+  const config = pluginData.config.get();
+  const reason = parseReason(config, formatReasonWithAttachments(args.reason, msg.attachments));
```

Comment:
Also, if `msg` is a Message from discord.js, this won't work. (`msg` is typed as any so I don't really know)

```suggestion
  const reason = formatReasonWithAttachments(parseReason(config, args.reason), msg.attachments);
```